articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -0,0 +1,218 @@
+
+     ![Create new key vault](images/azs-browser-create-key-vault.png)
+
+6. Click **Create**.
+
+7. You can monitor the progress of your Key Vault's deployment by clicking the **Notifications** icon.
+
+   ![!Notification showing vault deployment in progress](images/azsp_createvm_progress.png)
+
+### Adding a certificate to the Key Vault
+
+1. Once the key vault has been deployed, navigate to it by clicking **All services** in the favourites panel, then selecting **Key Vaults** under the *Security + Identity* section.
+
+2. Select your key vault from the list.
+
+3. In the **Settings** section of the key vault blade, select **Secrets**.
+
+4. On the **Secrets** page, click the **+ Generate/Import** button.
+
+5. In the **Create a secret** blade, enter the following information:
+
+   - **Upload Options** - Select the *Certificate* option.
+
+   - **Upload Certificate** - Select the .pfx certificate to upload.
+
+   - **Name** - The name of the certificate to identify it within the Key Vault.
+
+   - **Activation Date** - Sets when the certificate will become active.
+
+   - **Expiration Date** - Sets when the certificate will become inactive.
+
+   - **Enabled** -Set whether or not the secret data can be retrieved.
+
+   ![Create a new secret](images/azs-browser-add-certificate.png)
+
+6. Click **Create**.
+
+7. Repeat as necessary, depending on how many certificates you require.
+
+### Gathering Key Vault and certificate information
+
+During configuration of the Service Fabric cluster, several details relating to the Key Vault and certificates are required. It is also necessary to configure access to the key vault for Virtual Machines and the Azure Resource Manager.
+
+1. Once the key vault has been deployed, navigate to it by clicking **All services** in the favourites panel, then selecting **Key Vaults** under the *Security + Identity* section.
+
+2. Select your key vault from the list.
+
+3. In the **Settings** section of the key vault blade, select **Access Properties**.
+
+4. On the **Access policies** page, click **Click to show advanced access policies**.
+
+5. Tick the top two check boxes, then click the save button.
+
+6. In the **Settings** section of the key vault blade, select **Properties**.
+
+7. Copy the **Resource ID** and store it for later use.
+
+8. In the **Settings** section of the key vault blade, select **Secrets**.
+
+9. On the **Secrets** page, select the certificate you added in the previous section.
+
+10. On the certificate blade, select the current version.
+
+11. Copy the **Secret Identifier**.
+
+12. Repeat for each certificate that you are going to use for the service fabric cluster.
+
+The thumbprint of the certificate(s) is also required. A guide on how to find the thumbprint of the certificate(s) can be found [here](https://docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/how-to-retrieve-the-thumbprint-of-a-certificate).

```

Comment:
You also need the thumbprint of the certificate(s). For information about how to find this information, see [here](https://docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/how-to-retrieve-the-thumbprint-of-a-certificate).